test/test_onnx.py

```diff
@@ -0,0 +1,84 @@
+import numpy
+import io
+import torch
+from torchvision import ops
+
+# onnxruntime requires python 3.5 or above
+import sys
+if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] < 5):
+    sys.exit()
```

Comment:
I think the current idiom for skipping tests that are not Py2-compatible is to use `@unittest.skipIf(...)`, see for example https://github.com/pytorch/vision/blob/93bceaf250fe659f4baa9a1ecf79ece70d14b356/test/test_io.py#L57